misoc/integration/cpu_interface.py

```diff
@@ -187,7 +187,7 @@ def _get_rstype(size):
         return "u8"
 
 
-def _get_rw_functions_rs(reg_name, reg_base, size, nwords, busword, read_only, cpu_dw_bytes):
+def _get_rw_functions_rs(reg_name, reg_base, size, nwords, busword, read_only, cpu_dw_bytes, use_dmb=False):
```

Comment:
Since the behavior is potentially incorrect without ``dmb`` I don't think it should ever be disabled and the ``use_dmb`` parameter should be removed.

Reply:
Well, this is only Zynq-specific issue. There's no need for memory barriers for other platforms that don't use the simplified axi2csr bridge (and more importantly, don't have ``libcortex_a9`` or even ``dmb`` instructions available).